docs/platforms/aws-lambda/nodejs/index.md

```diff
@@ -0,0 +1,465 @@
+
+```javascript
+/* Tags: aws.dynamodb */
+
+aws.dynamodb.table_names: ["Users"]
+aws.dynamodb.segment: 10 // The value of the Segment request parameter.
+aws.dynamodb.total_segments: 100 // The value of the TotalSegments request parameter.
+aws.dynamodb.count: 5 // The value of the Count request parameter
+aws.dynamodb.scanned_count: 5 // The value of the ScannedCount request parameter.
+aws.dynamodb.attributes_to_get: ["lives", "id"] // The value of the AttributesToGet request parameter.
+aws.dynamodb.consistent_read: true // The value of the ConsistentRead request parameter.
+aws.dynamodb.index_name: "my_index" // The value of the IndexName request parameter.
+aws.dynamodb.limit: 50 // The value of the Limit request parameter.
+aws.dynamodb.projection: "Title, Description, RelatedItems, ProductReviews" // The value of the ProjectionExpression request parameter.
+aws.dynamodb.select: "ALL_ATTRIBUTES" // The value of the Select request parameter.
+```
+
+### `aws.sdk.dynamodb.updateitem`
+
+If you use the `aws-sdk` module in Node.js with AWS DynamoDB and perform this operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.dynamodb */
+
+aws.dynamodb.table_names: ["Users"]
+```
+
+### `aws.sdk.sqs.sendmessage`
+
+If you use the `aws-sdk` module in Node.js with AWS SQS and perform a `SendMessage` operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.sqs */
+
+aws.sqs.queue_name: "MyQueue" // Taken from the Queue URL
+aws.sqs.operation: "send" // Must be "send"
+aws.sqs.message_ids: ["fja98jafs"] // An array with the message ID provided in the SDK operation response.  Must be an array.
+```
+
+### `aws.sdk.sqs.sendmessagebatch`
+
+If you use the `aws-sdk` module in Node.js with AWS SQS and perform a `SendMessageBatch` operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.sqs */
+
+aws.sqs.queue_name: "MyQueue" // Taken from the Queue URL
+aws.sqs.operation: "send" // Must be "send"
+aws.sqs.message_ids: ["fja98jafs", ...] // An array with the message IDs provided in the SDK operation response.
+```
+
+### `aws.sdk.sns.publish`
+
+If you use the `aws-sdk` module in Node.js with AWS SNS and perform a `Publish` operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.sns */
+
+aws.sns.topic.name: "sns-lambda" // Taken from the TopicArn
+aws.sns.operation: "send" // Must be "send"
+aws.sns.message_ids: ["fja98jafs"] // Introspected from the response of this SDK operation
+```
+
+### `aws.sdk.sns.publishbatch`
+
+If you use the `aws-sdk` module in Node.js with AWS SNS and perform a `PublishBatch` operation these Tags are added to the Span:
+
+```javascript
+/* Tags: aws.sns */
+
+aws.sns.topic.name: "sns-lambda" // Taken from the TopicArn
+aws.sns.operation: "send" // Must be "send"
+aws.sns.message_ids: ["fja98jafs", ...] // Introspected from the response of this SDK operation
+```
+
+## `node.http`
+
+If you use the `http` module in Node.js, this Span is created.
```

Comment:
I assume  it's about `http.request` specifically, and we do not cover here usage of e.g. `http.createServer` 

Maybe then it'll better to name it `request` and in one of the properties store wether its _http_ or _https_ (?)

Reply:
That's right.  Though, I think we should name Spans after the module name first and foremost for explicitness and the clarity that follows.  Abstraction or clever terms will be too confusing ultimately.

Perhaps `node.http.request` is the right name here.